tests/benchmarks/serde-benchmarks/smithy-build.json

```diff
@@ -1,5 +1,6 @@
 {
   "version": "1.0",
+  "sources": ["model"],
```

Comment:
question: why doesn't this need to reference a `.smithy` file like the other smithy-build.json files?

Reply:
It can be a directory or file. https://smithy.io/2.0/guides/building-models/build-config.html